coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 100.0
+    "covered_percent": 99.66
```

Comment:
Looks like you should write some tests for this.

Reply:
yep